doc/migration-to-tt.rst

```diff
@@ -0,0 +1,51 @@
+Migration from Cartridge CLI to tt
+==================================
+
+To switch from Cartridge CLI to ``tt``
+
+To start managing a Cartridge application with ``tt`` instead of Cartridge CLI,
+run ``tt init`` in the application directory:
+
+.. code-block::bash
+
+    $ tt init
+    • Found existing config '.cartridge.yml'
+    • Environment config is written to 'tt.yaml'
+
+This creates a ``tt`` environment based on the existing Cartridge configuration.
+Now you're ready to manage the application with ``tt``:
+
+.. code-block::bash
+
+    $ tt start
+    • Starting an instance [app:s1-master]...
+    • Starting an instance [app:s1-replica]...
+    • Starting an instance [app:s2-master]...
+    • Starting an instance [app:s2-replica]...
+    • Starting an instance [app:stateboard]...
+    • Starting an instance [app:router]...
+    $ tt status
+    • app:router: RUNNING. PID: 13188.
```

Comment:
Current `status` output format is a table:
```
INSTANCE           STATUS          PID
app:s1-replica     RUNNING         112645
app:s2-master      RUNNING         112646
app:s2-replica     RUNNING         112647
app:stateboard     RUNNING         112655
app:router         RUNNING         112656
app:s1-master      RUNNING         112644
```

Reply:
Thanks, updated